### PR TITLE
Fix loop issue #54 (multiprocess python >3.8 on OSX).

### PR DESCRIPTION
**Pull request by DrSoulain**

### CODE REVIEW

**Comment by neutrinoceros** [comment]
I checked this in #55 and it seems that it does *not* close #54. However I'm fine with merging this as is because it doesn't seem to create more problems.

File: amical/externals/candid/candid.py [hunk 1]
```diff
@@ -1606,9 +1612,10 @@ def testInst(h):
                     ep = []
                     for i in range(len(data)):
                         if all(np.isnan(data[i])) or all(np.isnan(err[i])):
-                            tmp = {
-                                "A" + str(j): np.nan for j in range(reducePoly + 1)
-                            }, {"A" + str(j): np.nan for j in range(reducePoly + 1)}
+                            tmp = (
+                                {"A" + str(j): np.nan for j in range(reducePoly + 1)},
```
Comment (neutrinoceros) [inline amical/externals/candid/candid.py:1616]: I don't understand this change

Comment (DrSoulain) [reply amical/externals/candid/candid.py:1616]: It's auto-black formatting no?

Comment (neutrinoceros) [reply amical/externals/candid/candid.py:1616]: why is it coming up just now then ?

**Pull request merged**